.eslintrc.js

```diff
@@ -89,6 +115,30 @@ module.exports = {
         "no-console": "off",
       },
     },
+    {
+      files: ["./src/**/*.ts"],
```

Comment:
テストのファイルとかが抜けてるかもです。（node_modulesは無視されそうでした）
```suggestion
      files: ["*.ts"],
```

root直下のtsも見えるようになるので、tsconfig.jsonのincludeに↓の設定追加がいりそうでした
`"*.ts",`

Reply:
`tsconfig.json`に`*.ts`入れたらそれはそれで面倒になった記憶があります。とりあえず1段下を追加。

Reply:
試してみた感じあんまり問題にならなそうでした。
とはいえ対象は1ファイルだけなのでどっちでも良さそう。
ということでresolve！

ちなみにどういう時に面倒になる感じなんでしょ 👀

Reply:
この変更で`*/*.ts`が検知されなくなったことに気づきました 🙇 
新しくPR出します 🙏